aws/sdk/examples/snowball/src/bin/create-address.rs

```diff
@@ -70,34 +93,73 @@ async fn main() -> Result<(), aws_sdk_snowball::Error> {
         street1,
         street2,
         street3,
+        verbose,
     } = Opt::from_args();
 
     let region_provider = region::ChainProvider::first_try(region.map(Region::new))
         .or_default_provider()
         .or_else(Region::new("us-west-2"));
 
+    println!();
+
+    if verbose {
+        println!("Snowball version:       {}", PKG_VERSION);
+        println!(
+            "Region:                 {}",
+            region_provider.region().unwrap().as_ref()
+        );
+        println!("City:                   {}", city.clone());
```

Comment:
use `&city` (and equivalent) for all parameters here rather than cloning

Reply:
Will do